src/styles/index.scss

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  $minutes-diraction: 3600;
+  $seconds-diraction: 60s;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted black;
+  border-radius: 50%;
+
+  &--speed-up {
+    $minutes-diraction: 600s;
+    $seconds-diraction: 10s;
+  }
```

Comment:
The `&--speed-up` modifier does not change the CSS output because variable declarations inside a selector do not affect the variables used in the animations of the elements. You need to either use these variables directly in the animation properties or use another method to speed up the animations.